### PR TITLE
Pipboard: control the pipeline

### PR DESCRIPTION
**Pull request by utkobe**
Add several new features into pipboard

### CODE REVIEW

File: packages/cli/src/bin/pipcook-job.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ import { tunaMirrorURI } from '../config';
 const PipelineStatus = [ 'creating', 'running', 'success', 'fail' ];
 
 async function list(): Promise<void> {
-  const jobs = await get(`${route.job}/list`);
+  let jobs = await get(`${route.job}/list`);
+  jobs = jobs.rows;
   console.table(jobs.map((item: any) => {
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook-job.ts:16]: Using `jobs.rows` here to keep the `jobs` constant.

File: packages/cli/src/bin/pipcook-pipeline.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ import { get, post, put, del } from '../request';
 import { route } from '../router';
 
 async function list(): Promise<void> {
-  const pipelines = await get(`${route.pipeline}/list`);
+  let pipelines = await get(`${route.pipeline}/list`);
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook-pipeline.ts:10]:
```suggestion
  let pipelines = (await get(`${route.pipeline}/list`)).rows;
```

File: packages/daemon/src/runner/helper.ts [hunk 3]
```diff
@@ -15,19 +15,24 @@ import {
...
+      configJson = JSON.parse(await request(configPath));
+    } else if (urlObj.protocol === 'file:') {
+      configJson = await fs.readJSON(url.fileURLToPath(configPath));
+    } else {
+      throw new TypeError(`protocol ${urlObj.protocol} is not supported`);
+    }
   } else {
-    throw new TypeError(`protocol ${urlObj.protocol} is not supported`);
+    configJson = configPath;
   }
```
Comment (yorkie) [inline packages/daemon/src/runner/helper.ts:34]: Shall we have a new function called `loadConfig`?

File: packages/pipboard/ice.config.js [hunk 4]
```diff
@@ -20,4 +20,11 @@ module.exports = {
       target: 'http://127.0.0.1:6927',
     },
   },
+  define: {
+    CWD: JSON.stringify(path.join(__dirname, '..', '..')),
```
Comment (yorkie) [inline packages/pipboard/ice.config.js:24]:
```json
define: {
  ENV: process.env.DEV === 'TRUE' ? JSON.stringify('TRUE') : JSON.stringify('FALSE')
}
```

File: packages/cli/src/bin/pipcook-job.ts (hunk 1 above)
Comment (utkobe) [reply packages/cli/src/bin/pipcook-job.ts]: fixed

File: packages/cli/src/bin/pipcook-pipeline.ts (hunk 2 above)
Comment (utkobe) [reply packages/cli/src/bin/pipcook-pipeline.ts]: fixed

File: packages/daemon/src/runner/helper.ts (hunk 3 above)
Comment (utkobe) [reply packages/daemon/src/runner/helper.ts:35]: Yes, I modified and define a new func

File: packages/pipboard/ice.config.js (hunk 4 above)
Comment (utkobe) [reply packages/pipboard/ice.config.js:24]: Fixed

**Pull request merged**